gunicorn/config.py

```diff
@@ -1368,15 +1368,31 @@ class LogConfigDict(Setting):
     desc = """\
     The log config dictionary to use, using the standard Python
     logging module's dictionary configuration format. This option
-    takes precedence over the :ref:`logconfig` option, which uses the
-    older file configuration format.
+    takes precedence over the :ref:`logconfig` and :ref:`logConfigJson` options, which uses the
+    older file configuration format and JSON respectively.
 
     Format: https://docs.python.org/3/library/logging.config.html#logging.config.dictConfig
 
     .. versionadded:: 19.8
     """
 
 
+class LogConfigJson(Setting):
+    name = "logconfig_json"
+    section = "Logging"
+    cli = ["--log-config-json"]
+    meta = "FILE"
+    validator = validate_string
+    default = None
+    desc = """\
+    The log config JSON reads config from a JSON file
+
+    Format: https://docs.python.org/3/library/logging.config.html#logging.config.dictConfig
+
+    .. versionadded:: 19.9
```

Comment:
I think "19.9" should be "20.0" now.

Reply:
is there a plan to upgrade to 20.0? The current is 19.9 and this could be going to either 19.10 or 20.0.

Reply:
The master branch doesn't support Python 2 anymore and since we can't drop Python 2 support in the next 19.x release, we can safely say that the master branch is now 20.0. If you want to update `gunicorn.__version__`, feel free to submit a separate PR :)